client/jetpack-onboarding/steps/stats.jsx

```diff
@@ -107,7 +107,7 @@ class JetpackOnboardingStatsStep extends React.Component {
 		const { activatedStats, getForwardUrl, siteId } = this.props;
 
 		return (
-			<div className="steps__main">
+			<div className="steps__main" data-e2e-type="activate-stats">
```

Comment:
Is there any reason for this `data-e2e-type` to be different from the step name (`stats`)?

Reply:
No reason, I mapped this to the e2e test `ActivateStatsPage` class. To stay consist with calypso I'll send in a commit updating this and the e2e tests to use `stats`.

Reply:
Thank you 👍